dolby_vision/src/rpu/generate.rs

```diff
@@ -26,11 +26,16 @@ pub struct GenerateConfig {
     pub cm_version: CmVersion,
 
     /// Profile to generate
+    ///  - 5: IPT base layer with no reshaping
     ///  - 8.1: HDR10 base layer
     ///  - 8.4: HLG base layer with static reshaping (iPhone 13 MMR)
     #[cfg_attr(feature = "serde_feature", serde(default))]
     pub profile: GenerateProfile,
 
+    /// Set scene cut flag for every frame.
+    #[cfg_attr(feature = "serde_feature", serde(default))]
+    pub long_play_mode: bool,
```

Comment:
This would override the config since clap defaults flags to false.
It should be an Option like profile.